test/index.ts

```diff
@@ -0,0 +1,115 @@
+import tape from "fresh-tape";
+
+import { Member } from "../src/member";
+
+const EXAMPLE_ID = "example";
+
+tape("Able to initialize a member", (t) => {
+  const member = new Member();
+
+  member.processFeeds();
```

Comment:
Shouldn't processFeeds() be part of the syncing process? What advantage do we get of separating those?

Reply:
Main thing here was the auto-accept function meant that processing feeds would trigger member adds, and I was hoping to have more control of when that happened. I can refactor to work around that so that control of accepting could be outside of syncing feeds.

Other consideration was that I wanted a place for members to process requests and responses that they generated themselves 1. for the initial member case and 2. to remove the need for extra logic for that bit. Down to add proccesFeeds at the end of sync, though.

Reply:
> I can refactor to work around that so that control of accepting could be outside of syncing feeds.

Structurally it is probably for the best, though I don't think we need to use manual confirmation in our first real scenario. The reason why I brought this up is because in the sync process it may be possible to avoid some chance of forks by having one member sync first and process the entries before syncing everything - including the need entries - with the other peer.

Reply:
Yeah, I'm thinking that I'll have a `getPendingRequests()` method and a `acceptPendingRequests()`